Added 5 Solutions (#56)
priyanshisharma/Leetcoding-Challenge: April/README.md

# April-Leetcoding-Challenge

> ### This repository contains the solutions I submitted during the June Leetcoding Challenge. Below are the links of questions on LeetCode if you wish to try them yourself!

Reviewer: ```suggestion
### This repository contains the solutions for the April Leetcoding Challenge. Below are the links of questions on LeetCode if you wish to try them yourself!
```
